### ISSUE TITLE
Get labels for full training set

### ISSUE THREAD

**Issue by wmorning** [jcpeterson/cifar-10h#1]
Hi Joshua!

Thanks for making this dataset publicly available.  We think this may be a cool dataset to look at in some experiments that we and probably others are working on.  To that end, we were wondering if you plan to do a similar human labeling process to get labels for the full training set.  This may help in setting up CIFAR-10h for use as a standard benchmark to test model calibration and reliability.

We look forward toward experimenting with this dataset, and of course, citing your paper :)

Best,
Warren

**Comment by jcpeterson** [comment]
@wmorning, I'm glad you're finding it useful. We are indeed considering collecting labels for the training set. We also have another set of human-derived labels for the test set that will hopefully be released this year.
